libs/designsystem/src/lib/components/toast/services/toast.helper.spec.ts

```diff
@@ -79,10 +79,24 @@ describe('ToastHelper', () => {
           await TestHelper.whenReady(ionToast);
           const toastWrapper = ionToast.shadowRoot.querySelector('.toast-wrapper');
           expect(toastWrapper).toHaveComputedStyle({
-            'background-color': getColor(notificationColor),
+            'background-color':
+              messageType !== 'danger' ? getColor(notificationColor) : getColor('warning'),
           });
         });
       });
+
+      it('Danger type should display warning in console', async () => {
```

Comment:
Consider wording (tests reads like: "it should xxx....")
```suggestion
      it(`should display warning in console when using 'danger' MessageType`, async () => {
```